SpatialOS/scripts/generate-schema-descriptor.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+set -e -x
+pushd "../"
+
+BUILD_DIR="$1"
+PACKAGES_DIR="$2"
+
+SCHEMA_DIR="${BUILD_DIR}"/SpatialOS/schema
+SCHEMA_BIN_DIR="${BUILD_DIR}"/SpatialOS/schema_out/bin
+OUT_DIR="${SCHEMA_BIN_DIR}"/generated/csharp
+
+# Generate C# code from the schemas:
```

Comment:
This part here generates C# code which is unrelated to the schema descriptor. This part should be in the worker specific build.sh files.

Reply:
done